README.md

```diff
@@ -36,12 +36,8 @@ commands for the respective tasks.
 Deploying the site to [GitHub pages](https://remotestorage.github.io/website/)
 for testing/review:
 
-    ./deploy
-
-This currently requires [git-up](https://github.com/aanand/git-up) to be
-installed. You can comment the line, if you want. Make sure you're up-to-date
-on all branches yourself then.
+	./deploy
 
 Deploying to production (need 5apps remote set, see comment in deploy script):
 
-    ./deploy production
+	./deploy production
```

Comment:
What's the reason for changing the indentation to more than 4 spaces? We use 4 spaces in all remoteStorage repos and documents for this.

Reply:
Yeah sorry, forgot to add this repo to the list of projects that Emacs shouldn't use tabs for. I'll get that changed back